### PR TITLE
review fixes, folder structure and table fixes

### PR DESCRIPTION
**Pull request by Sylpherena**

### CODE REVIEW

File: src/providers/theme/ThemeProvider.tsx [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import React, { useEffect, useState } from "react";
-import { DaisyUIThemes } from "../../themes";
+import { daisyUIThemes, DaisyUIThemes } from "./themes";
```
Comment (hsynaksu) [inline src/providers/theme/ThemeProvider.tsx]: DaisyUITheme instead of DaisyUIThemes

File: src/pages/Applications/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import List from "./List";
+
+export default function Applications() {
```
Comment (hsynaksu) [inline src/pages/Applications/index.tsx]: index files should only contain exports not whole components

File: src/pages/Applications/List/Add/Form.tsx [hunk 3]
```diff
@@ -1,88 +1,78 @@
 import { FieldValues, useForm } from "react-hook-form";
-import InputWithLabel from "../../components/InputWithLabel";
+import InputWithLabel from "../../../../components/InputWithLabel";
 import { zodResolver } from "@hookform/resolvers/zod";
-import { formatDate, formSchema } from "./addFormSchema";
+import { defaultFormValues, formSchema } from "./formSchema";
 
-function AddForm() {
+export default function Form() {
```
Comment (hsynaksu) [inline src/pages/Applications/List/Add/Form.tsx]: Add feature can be moved to Application folder

File: src/pages/Applications/List/Add/ModalButton.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            className="btn btn-sm btn-circle absolute right-2 top-2"
+            onClick={toggleModal}
+          >
+            ✕
+          </button>
+          <h3 className="font-bold text-lg">Add New Application</h3>
+          <AddForm />
+        </div>
+        <form method="dialog" className="modal-backdrop">
+          <button onClick={toggleModal}>close</button>
```
Comment (hsynaksu) [inline src/pages/Applications/List/Add/ModalButton.tsx]: this should only close the modal, not toggle it

Comment (Sylpherena) [reply src/pages/Applications/List/Add/ModalButton.tsx]: done

File: src/pages/Applications/List/Add/Form.tsx (hunk 3 above)
Comment (Sylpherena) [reply src/pages/Applications/List/Add/Form.tsx]: done

File: src/pages/Applications/index.tsx (hunk 2 above)
Comment (Sylpherena) [reply src/pages/Applications/index.tsx]: done

File: src/providers/theme/ThemeProvider.tsx (hunk 1 above)
Comment (Sylpherena) [reply src/providers/theme/ThemeProvider.tsx]: done

**Review by hsynaksu — APPROVED** [review]

**Pull request merged**